control-plane/agents/src/bin/ha/node/server.rs

```diff
@@ -273,6 +301,26 @@ fn parse_uri(new_path: &str) -> Result<ParsedUri, SvcError> {
     ParsedUri::new(uri)
 }
 
+/// Gets all Nvme subsystems registered for a given volume.
+fn get_subsystems(parsed_uri: &ParsedUri) -> Result<Vec<NvmeCtrller>, SvcError> {
```

Comment:
@tiagolobocastro , I think this has to go into mayastor-dependency. Added this here as of now as i dont have merge access. Will raise new PR in that repo.

Reply:
IIRC @Abhinandan-Purkait already added this functionality, or similar?

Reply:
Its slightly different, this method will return list(if applicable) of NvmeSubsystems based on nqn. Whereas what @Abhinandan-Purkait has added will filter using address and nqn and return the First Subsystem that matches.

Reply:
@tiagolobocastro . I have raised a separate for moving this function to mayastor-dependency repo.